MonMaraicher/Features/FarmerDetail/FarmerDetailsView.swift

```diff
@@ -36,20 +37,22 @@ struct FarmerDetailsView: View {
                         .clipShape(.rect(cornerRadius: 16))
```

Comment:
Why you didn't add these modifiers in the property declaration?

Reply:
I wanted to keep the most important changes here. But your suggestion is not bad